modules/efk-logging-eventrouter-deploy.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-eventrouter.adoc
+
+[id='efk-logging-eventrouter-deploy_{context}']
+= Deploying and Configuring the Event Router
+
+Use the following steps to deploy Event Router into your cluster.
+
+.Prerequisites
+
+. Create a Service Account for the Event Router
++
+[source,yaml]
+----
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: cluster-logging-eventrouter
+  namespace: openshift-logging <1>
+----
+<1> Specify the namespace where eventrouter is deployed. 
+
+. Create a ClusterRole to monitor for events in the cluster.
++
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: event-reader
+rules:
+- apiGroups: [""]
+  resources: ["events"] <1>
+  verbs: ["get", "watch", "list"] <2>
+----
+<1> Specify the `events` resource.
+<2> Specify the `get`, `watch`, and `list` permissions. 
+
+. Create a ClusterRoleBinding to bind the ClusterRole to the ServiceAccount.
++
+[source,yaml]
+---- 
+kind: ClusterRoleBinding
+apiVersion: v1
+metadata:
+  name: event-reader-binding
+subjects:
+- kind: ServiceAccount
+  name: cluster-logging-eventrouter <1>
+roleRef:
+  kind: ClusterRole
+  name: event-reader <2>
+  namespace: openshift-logging <3>
```

Comment:
Since it is a ClusterRoleBinding, we don't need namespace here?

Reply:
@nhosoi So, the template creates the service account, cluster role, and cluster role bindings? 
Did I misread @jcantrill README file:
_Elevated permissions are required to create the cluster level roles and bindings referenced in the following template_ 
That doesn't mean the template references existing CR and CRB?

Reply:
Hmmm, I thought you need to have an enough privilege to run "oc process -f <templatefile> | oc apply -f -"...  In my testing, I ran the command line with all-in-one <templatefile>, which made the eventrouter pod set up correctly... (note: without any prerequisites)
But yes, I think we'd better have an input from @jcantrill. ;)  Thanks!

Reply:
@nhosoi I removed the prereqs to create the service account, cluster role, and binding. Add ed a line about needing permission to update in `openshift-*` namespace, per @ewolinetz [comment in Slack](https://coreos.slack.com/archives/CB3HXM2QK/p1550874627165300).

Reply:
@mburke5678, thanks!
/lgtm